ethereum/contracts/Splitter.sol

```diff
@@ -1,17 +1,14 @@
 pragma solidity >=0.5.8;
 
-contract Splitter {
+import './Pausable.sol';
+
+contract Splitter is Pausable {
   mapping(address => uint) accounts;
-  address owner;
 
   event SplitHandle(address from, address receiver1, address receiver2);
```

Comment:
Suggested naming convention, LogSplit, LogWithdrawal. Corresponds to the functions. Distinguishable from constructors, structs, enums, contracts that also use CapsCase.